src/components/Navbar.vue

```diff
@@ -5,8 +5,8 @@
         </div>
         <ul class="nav-links">
             <li><a href="/">Home</a></li>
-            <li><a href="#about">About</a></li>
-            <li><a href="#about-us">About us</a></li>
+            <li><a href="/#about">About</a></li>
+            <li><a href="/#about-us">About us</a></li>
```

Comment:
My idea for the About section and about us would be the about would describe the game and other game related information, while the about us would be about our team. Perhaps It would be best to change the About title to something more descriptive, although Im not sure what I would put. Maybe "How to play" or "About Game"? I'll have to think about it.

Reply:
How about `About the Game` and `Meet the Team`? Something like that?